server/cmd/dcrdex/main.go

```diff
@@ -119,17 +119,21 @@ func mainCore(ctx context.Context) error {
 		fmt.Printf("Load swapper state from file %q with time stamp %v? (y, n, or enter to abort) ",
 			stateFile.Name, encode.UnixTimeMilli(stateFile.Stamp))
 		scanner := bufio.NewScanner(os.Stdin)
-		scanner.Scan()
+		scan := make(chan bool, 1)
+		go func() {
+			scan <- scanner.Scan()
+		}()
+		// Wait for EOF or context done.
```

Comment:
Actually not EOF but a line on the scanner ending with the newline (default token for `Scanner`).